### PR TITLE
docs(tem): webhook events payload

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+meta:
+  title: Understanding Webhook event payloads
+  description: This page details the fields of the webhook event payloads for Scaleway Transactional Email
+content:
+  h1: Understanding Webhook event payloads
+  paragraph: This page details the fields of the webhook event payloads for Scaleway Transactional Email
+tags: spf spf-record dkim dkim-records dmarc dmarc-record transactional
```
Comment (jcirinosclwy) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:8]:
```suggestion
tags: spf dkim dmarc transactional
```

**Review by jcirinosclwy — APPROVED** [review]

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+## email_queued
+```json
+{
+	"id": string,
+	"type": "email_queued",
+ 	"organization_id": string,
+	"project_id": string,
+    "domain_id": string,
+	"domain_name": string,
+    "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:54]:
```suggestion
    "created_at": date,
```

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	"email_queued_at": date,
+    "email_id": string,
+    "email_from": string,
+	"email_to": string,
+	"email_headers": [{"name": string, "value": string}],
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:59]: We should put the slice or array in multiple lines, or the github view messed up with the colors

Like this:
```suggestion
	"email_headers": [{
		"name": string,
		"value": string
	}],
```

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+## email_dropped
+```json
+{
+	"id": string,
+	"type": "email_dropped",
+	"organization_id": string,
+	"project_id": string,
+    "domain_id": string,
+	"domain_name": string,
+    "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:71]:
```suggestion
    "created_at": date,
```

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+|---------|-------------------------------------|
+| `id`     | Event identifier                                                                          |
+| `type`     | Event type                                      |
+| `organization_id`    | Domain Organization                            |
+| `project_id` | Domain Project                                                       |
+| `domain_id`  | Domain identifier                                                       |
+| `domain_name`   | Domain name  |
+| `created_at`    | Event creation date                            |
+| `email_sent_at`    | 	Email reception date by the recipient server                          |
+| `email_queued_at` | Email reception date in by TEM proxy or API                                                          |
```
Comment (moutoum) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:31]:
```suggestion
| `email_queued_at` | Email reception date by TEM proxy or API                                                          |
```

File: managed-services/transactional-email/api-cli/webhook-events-payload.mdx [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+| `email_id`  | Email identifier (used for GetEmail endpoint)                                                      |
+| `email_from`   | Email FROM address |
+| `email_to`  | Email recipient address                                                          |
+| `email_headers`   | SMTP headers  |
+| `email_error`   | SMTP message explaining the drop  |
+| `email_next_try`   | Next time we will try to send the email to the recipient server  |
```
Comment (moutoum) [inline managed-services/transactional-email/api-cli/webhook-events-payload.mdx:37]:
```suggestion
| `email_next_try`   | Next time the email will be sent to the recipient server  |
```

**Review by moutoum — CHANGES_REQUESTED** [review]
For date types, I'd suggest to had the date format: `RFC3339`

**Review by nerda-codes — APPROVED** [review]

File: managed-services/transactional-email/api-cli/webhook-events-payloads.mdx [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+## email_delivered
+```json
+{
+  "id": string,
+  "type": "email_delivered",
+  "organization_id": string,
+  "project_id": string,
+  "domain_id": string,
+  "domain_name": string,
+  "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payloads.mdx:99]:
```suggestion
  "created_at": date,
```

File: managed-services/transactional-email/api-cli/webhook-events-payloads.mdx [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+## email_mailbox_not_found
+```json
+{
+  "id": string,
+  "type": "email_mailbox_not_found",
+  "organization_id": string,
+  "project_id": string,
+  "domain_id": string,
+  "domain_name": string,
+  "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payloads.mdx:121]:
```suggestion
  "created_at": date,
```

File: managed-services/transactional-email/api-cli/webhook-events-payloads.mdx [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+## email_spam
+```json
+{
+  "id": string,
+  "type": "email_spam",
+  "organization_id": string,
+  "project_id": string,
+  "domain_id": string,
+  "domain_name": string,
+  "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payloads.mdx:143]:
```suggestion
  "created_at": date,
```

File: managed-services/transactional-email/api-cli/webhook-events-payloads.mdx [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+## email_deferred
+```json
+{
+  "id": string,
+  "type": "email_deferred",
+  "organization_id": string,
+  "project_id": string,
+  "domain_id": string,
+  "domain_name": string,
+  "created_at": string,
```
Comment (jerjako) [inline managed-services/transactional-email/api-cli/webhook-events-payloads.mdx:165]:
```suggestion
  "created_at": date,
```

**Review by jerjako — APPROVED** [review]

**Pull request merged**